cmd/scollector/collectors/interval.go

```diff
@@ -111,6 +112,10 @@ func enableURL(url string, regexes ...string) func() bool {
 			return false
 		}
 		defer resp.Body.Close()
+		if resp.StatusCode != 200 || len(res) == 0 {
```

Comment:
Could just batch this up in an anonymous defer func rather than having this awkward condition here that mirrors future conditions. Since we ignore errors intentionally from `CopyN` in this case, doesn't matter if a `ReadAll` has already occurred.

Just a nitpick, tho. Fine as is if you'd prefer.